### ISSUE TITLE
[Installer] vendor/bin directory issues

### ISSUE THREAD

**Issue by willrowe** [laravel/framework#9051]
The other day when trying to run PHPUnit on a fresh install from the Laravel Installer I was met with `vendor/bin/phpunit: Permission denied` and sure enough, after looking at the `vendor/bin` directory I found this:
```
-rw-r--r-- 527  Jun  3 20:41 classpreloader.php
-rw-r--r-- 750  Jun  3 20:41 phpspec
-rw-r--r-- 930  Jun  3 20:41 phpunit
-rw-r--r-- 3296 Jun  3 20:41 psysh
```
I then deleted the `vendor` directory and ran `composer install`. After that finished I went back and looked at the `vendor/bin` directory and found this:
```
lrwxr-xr-x 51 Jun  3 20:45 classpreloader.php -> ../classpreloader/classpreloader/classpreloader.php
lrwxr-xr-x 30 Jun  3 20:45 phpspec -> ../phpspec/phpspec/bin/phpspec
lrwxr-xr-x 26 Jun  3 20:45 phpunit -> ../phpunit/phpunit/phpunit
lrwxr-xr-x 22 Jun  3 20:45 psysh -> ../psy/psysh/bin/psysh
```
So it appears that there are two issues when the files are being zipped:
1 - The executable permissions aren't being preserved (at the very least it should be on for the owner).
2 - The symlinks aren't being preserved. It appears that the contents of the files match the files they're supposed to link to, but they should be symlinks all the same.

**Comment by GrahamCampbell** [comment]
Please ask on the forums.

**Comment by willrowe** [comment]
This is a bug that needs to be addressed. I have layed out exactly what the problem is and a possible solution. Since it's a problem with the zip file I cannot submit a PR, I tried to call attention to this on IRC to no avail. If this is something that is happening just for me then that's one thing, but I haven't been able to get an answer either way. I'd greatly appreciate if someone could confirm whether or not this is happening for them too.
